mettle/configure.ac

```diff
@@ -19,17 +19,17 @@ case $host_os in
 		HOST_OS=apple
 		OBJCFLAGS="$OBJCFLAGS -fobjc-arc"
 		AC_DEFINE(HAVE_WEBCAM)
-		AC_SUBST([PLATFORM_LDADD], ['-framework CoreMedia -framework CoreImage -framework AVFoundation -framework ImageIO'])
+		AC_SUBST([PLATFORM_LDADD], ['-framework CoreMedia -framework QuartzCore -framework AVFoundation -framework ImageIO'])
 		AC_EGREP_CPP(yes,
 			[#import <TargetConditionals.h>
-			 #if TARGET_OS_OSX
+			 #ifndef TARGET_OS_OSX
 			 yes
 			 #endif],
```

Comment:
I wonder if the lexer is treating the  `],` as part of the program
https://github.com/bcook-r7/mettle/pull/9 seems to work?